backend/sozisel/lib/sozisel/media_storage/disk.ex

```diff
@@ -32,9 +32,22 @@ defmodule Sozisel.MediaStorage.Disk do
   end
 
   defp full_file_path(filename) do
-    Path.join([
-      Application.fetch_env!(:sozisel, SoziselWeb.Recordings) |> Keyword.fetch!(:upload_path),
-      filename
-    ])
+    filename
+    |> String.split(".")
```

Comment:
You know that there is this `Path` module that can extract extensions?

Reply:
Yeah, that true.